editions/tw5.com/tiddlers/filters/syntax/Filter Whitespace.tid

```diff
@@ -1,11 +1,15 @@
 created: 20150124182304000
-modified: 20150125105243000
-tags: [[Filter Syntax]]
+modified: 20230315204932346
+tags: [[Filter Expression]]
 title: Filter Whitespace
 type: text/vnd.tiddlywiki
 
+Parents: <$macrocall $name=".recursiveParentTag" current=<<currentTiddler>> stop="Filters"/>
+
 <$railroad text="""
+\start none
+\end none
 {( "space" | "tab" | "linefeed" | "return" | "vertical tab" | "formfeed" )}
 """/>
 
-Whitespace characters can appear between each [[run|Filter Run]] of a [[filter expression|Filter Expression]].
+Whitespace characters can appear between each run of a [[filter expression|Filter Expression]].
```

Comment:
We should not remove the first link to an explanatory tiddler (here "Filter Run") in the text. After all, a good wiki depends on the interlinking of its content.

Reply:
It should link to "Filter Expression" only, where the "whitespace" link comes from. At the "Filter Run" tiddler, there is no whitespace, because it's at Filter Expression.